MekHQ/src/mekhq/campaign/ResolveScenarioTracker.java

```diff
@@ -1192,7 +1194,13 @@ private void loadUnitsAndPilots(final @Nullable File unitFile) {
             } else {
                 // Enemy crew/pilot entity is actually in the salvage list
                 if (e instanceof EjectedCrew && null != e.getCrew() && !e.getCrew().getExternalIdAsString().equals("-1")) {
-                    enemyEjections.put(UUID.fromString(e.getCrew().getExternalIdAsString()), (EjectedCrew)e);
+                    // check for possible traitors
+                    if (scenario.isTraitor(UUID.fromString(e.getCrew().getExternalIdAsString()))) {
+                        pilots.put(UUID.fromString(e.getCrew().getExternalIdAsString()), e.getCrew());
+                        ejections.put(UUID.fromString(e.getCrew().getExternalIdAsString()), (EjectedCrew)e);
```

Comment:
```suggestion
                        ejections.put(UUID.fromString(e.getCrew().getExternalIdAsString()), (EjectedCrew) e);
```

Reply:
Fixed this in several other places in that method as well where it pre-existed this PR.